subgraph/pipelines/alpha-calibnet.yaml

```diff
@@ -3,10 +3,10 @@ version: 6
 resource_size: s
 apiVersion: 3
 sources:
-  source__fwss__data_set_created:
+  filbeam__data_set_created:
     type: subgraph_entity
     subgraphs:
-      - name: fwss-alpha-calibnet
+      - name: filbeam-alpha-calibnet
```

Comment:
I will call the hand-written subgraph "filbeam-alpha-calibnet", because it's reading events from multiple contracts (PDPVerifier, FWSS).